imageHover.js

```diff
@@ -0,0 +1,9 @@
+function bigImg(x) {
+  x.style.height = "250px";
```

Comment:
Method names should signify if they are altering the state of an object or if they are just returning an object. In this case, those 2 methods are altering the height and width of the image and it should be clear, something like `makeImageBig` or `applyBigImageStyling`. Also do not use variable names like `x` use a self explanatory name instead, like `image`